ci: snap: can't modify default configuration file

```
INFO: Enabling kata tracing on /snap/kata-containers/x1/usr/share/defaults/kata-containers/configuration.toml
[Errno 30] Read-only file system: '/snap/kata-containers/x1/usr/share/defaults/kata-containers/configuration.toml'
jaeger
jaeger
INFO: Disabling kata tracing on /snap/kata-containers/x1/usr/share/defaults/kata-containers/configuration.toml
[Errno 30] Read-only file system: '/snap/kata-containers/x1/usr/share/defaults/kata-containers/configuration.toml'
Makefile:147: recipe for target 'tracing' failed
make: *** [tracing] Error 1
Build step 'Execute shell' marked build as failure
```

--- a/integration/containerd/cri/integration-tests.sh
+++ b/integration/containerd/cri/integration-tests.sh
@@ -29,6 +29,7 @@ readonly runc_runtime_bin=$(command -v "runc")
 readonly CRITEST=${GOPATH}/bin/critest
 
 # Flag to do tasks for CI
+SNAP_CI=${SNAP_CI:-""}
 CI=${CI:-""}
 
 # Default CNI directory
@@ -77,8 +78,13 @@ ci_cleanup() {
 	fi
 
 	ID=${ID:-""}
-	if [ "$ID" == ubuntu ] &&  [ -n "${CI}" ] ;then
-		[ -f "${kata_config}" ] && sudo rm "${kata_config}"
+	if [ "$ID" == ubuntu ]; then
+		if [ -n "${SNAP_CI}" ]; then
+			# restore default configuration
+			sudo cp "${default_kata_config}" "${kata_config}"
+		elif [ -n "${CI}" ] ;then
+			[ -f "${kata_config}" ] && sudo rm "${kata_config}"
+		fi
 	fi
 }
 

